plugins/catalog-backend-module-ldap/report.api.md

```diff
@@ -24,6 +24,12 @@ import { SearchOptions } from 'ldapjs';
 import { UserEntity } from '@backstage/catalog-model';
 import { UserTransformer as UserTransformer_2 } from '@backstage/plugin-catalog-backend-module-ldap';
 
+// @public
+export const ActiveDirectoryVendor: LdapVendor;
```

Comment:
Mmmm. If we are making these part of the public interface, could we at least make it like

```ts
export const vendors = {
  activeDirectory: ... as LdapVendor,
  default: ... as LdapVendor,
} as const;
```

or so? That way they are all collected in a struct neatly together in the public interface instead of being all over the place

Reply:
Agree. It would keep the public interface cleaner